<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTare">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tare</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIngriedient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxIngridient">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Unit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All Purpose flour (dip and sweet method)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AP flour (spooned into the cup)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AP flour (sifted, then measured)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bread flour (dip and sweep)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bread flour (spooned into the cup)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bread flour (sifted, then measured)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cake flour (dip and sweep method)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cake flour (spooned into the cup)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cake flour (sifted, then measured)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>whole wheat flour (dip and sweep)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>whole wheat flour (sifted, then measured)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cornmeal (dip and sweep)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cornstarch (lightly spooned or sifted)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>butter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>clarified butter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vegetable shortening</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vegetable oil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>canola oil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>safflower oil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>heavy cream</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sour cream</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>full-fat yogurt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>buttermilk</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>whole milk</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>half and half</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cr√®me fraiche</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mascarpone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>water</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>granulated sugar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dark brown sugar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>light brown sugar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>powdered sugar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>honey</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>molasses</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>corn syrup</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxMeasurementType">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Measurements</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>US</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Metric</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imperial</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
